app/presenters/account_reset/pending_presenter.rb

```diff
@@ -19,8 +19,41 @@ def time_remaining_until_granted(now: Time.zone.now)
       )
     end
 
-    def account_reset_deletion_period_hours
-      IdentityConfig.store.account_reset_wait_period_days.days.in_hours.to_i
+    def account_reset_deletion_period
+      current_time = Time.zone.now
+
+      distance_of_time_in_words(
+        current_time,
+        current_time + account_reset_wait_period_days,
+        true,
+        accumulate_on: reset_accumulation_type,
+      )
+    end
+
+    def account_reset_wait_period_days
+      if supports_fraud_account_reset?
+        IdentityConfig.store.account_reset_fraud_user_wait_period_days.days
+      else
+        IdentityConfig.store.account_reset_wait_period_days.days
+      end
+    end
+
+    def supports_fraud_account_reset?
+      (account_reset_request_user.fraud_review_pending? ||
+        account_reset_request_user.fraud_rejection?) &&
+        IdentityConfig.store.account_reset_fraud_user_wait_period_days.present?
+    end
+
+    def account_reset_request_user
+      account_reset_request.user
+    end
+
+    def reset_accumulation_type
+      if account_reset_wait_period_days > 3
```

Comment:
The unitless `3` here causes this to become `true` even when it's 1 day. Can we use `AccountResetConcern` to avoid issues from inconsistencies in duplicated code? If we can't use a concern here, could we convert it into a plain service module instead?

Note "1 day waiting period" in the screenshot:

![Screenshot 2024-03-06 at 8 19 28 AM](https://github.com/18F/identity-idp/assets/1779930/7134e73b-548c-4bea-97b1-48c4852bf23b)